src/frontend/src/components/LoginContainer/Form/Form.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Form.css';
+import Button from './Button/Button';
+
+const Form: React.FC = () => {
+  return (
+    <div className="Form">
+      <form className="Login">
+        <div className="form-group">
+          <p> Sign In </p>
+          <div className="e-mail">
+            <p className="info">E-Mail</p>
+            <input type="text" name="e-mail" />
+          </div>
+          <div className="password">
+            <p className="info">Password</p>
+            <input type="password" name="pass" />
+          </div>
+          <label>
+            <Button />
```

Comment:
If you're not passing any props to this, it doesn't need to be a separate component, unless you expect custom functionality in the future.